components/Sidebar/SidebarBehavior.ts

```diff
@@ -32,14 +32,15 @@ export class SidebarBehavior {
 
     onResize?: NavigationResized;
 
-    constructor(onResize?: NavigationResized) {
+    constructor(collapse: boolean, onResize?: NavigationResized) {
         // Needed elements:
         this.body = $('body');
         this.nav = $('.Nav');
         this.navHandle = $('.NavExpander');
+        this.isCollapsed = collapse;
 
         // Interaction variables
-        this.isCollapsed = 'true' === getStored('jdl.navbar.collapsed');
+        this.isCollapsed = collapse;
```

Comment:
I think it should be:

```
this.isCollapsed = collapse !== undefined ? collapse : 'true' === getStored('jdl.navbar.collapsed');
```
You don't want to overrite (from the prop) if the user has already interacted with the sidebar (was collapsed or expanded), we should probably rename  `collapse` to `collapsedByDefault`.